topdown/http.go

```diff
@@ -95,6 +97,8 @@ func executeHTTPRequest(bctx BuiltinContext, obj ast.Object) (ast.Value, error)
 		} else if key == "url" {
 			url = obj.Get(val).String()
 			url = strings.Trim(url, "\"")
+		} else if key == "enableRedirect" {
+			enableRedirect, err = strconv.ParseBool(obj.Get(val).String())
```

Comment:
ℹ️ If we don't care about the `err` (which could be the case, since if `err != nil`, this also means `enableRedirect == false`), we could do `enableRedirect, _ = strconv.ParseBool(obj.Get(val).String())`